src/css/parse.js

```diff
@@ -2834,7 +2834,7 @@ function getUrange() {
   const column = token.col;
   let content = [];
 
-  pos += tokens[startPos].urangeEnd;
+  pos += tokens[startPos].urangeEnd - startPos;
   content = joinValues(startPos, tokens[startPos].urangeEnd);
   pos++;
```

Comment:
Can't you remove line 2837 completely and here make `pos = tokens[startPos].urangeEnd) + 1`?